_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
+- [Endorsement](#Endorsement)
+- [Outreach](#outreach)
+- [Feedback](#feedback)
+
+### Endorsement
+
+To be official, strategies, actions and other initiatives, must be endorsed or approved by management (DG, CIO, DM, ..).
```

Comment:
Do we?

The other option would be if we get approval for Denis to have delegated authority to send quarterly strategies, then when Denis sends our a quarterly strategy that could be approval for the strategies associated with the actions, no?

More lean approach, no?

Reply:
Things need to be approved.

Add it as another option: CIOEC, BN and Delegated authority.  But even if DG can sent direction, it probably still need to be presented to DG Comm and CIOEC. At least for information.

Reply:
Agree with @smellems , for some strategies, we may even have to go further up than ADMO. Technically, departmental strategies requires PMB approval. We're taking a non-conventional method of doing bite-size strategies. 

But ultimately, I think we should have a ["overall strategy"](https://sara-sabr.github.io/ITStrategy/esdc-it-strategy.html) that should be presented at CMC (and for that, it needs to be endorsed by ADMO)

Reply:
Post conversation with @remyb2canada, perhaps the issue here is "strategy" is too broad a term.

There may be no "strategy" approval process, it would depend on the size and implications of said strategies. 

Perhaps small iterative quarterly strategies are delegated approval from CIO to DG of strategy, while other larger and more impactful strategies need approval, or perhaps some smaller strategies (like the comms strategy itself) need only be approved by our director.

If we do not take this approach any quarterly strategy at all is optimistic if we have the follow the same process as we would a strategic for an enterprise transformational strategy